From cf0eebe267aad16d610498b4578c35fbfd349cb7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matthias=20Mail=C3=A4nder?=
 <Mailaender@users.noreply.github.com>
Date: Sun, 26 Jun 2022 11:24:23 +0200
Subject: [PATCH 1/3] Fix stock icon is not valid

<icon> not allowed in desktop appdata
---
 io.github.naikari.Naikari.metainfo.xml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/io.github.naikari.Naikari.metainfo.xml b/io.github.naikari.Naikari.metainfo.xml
index e2968f1600..dbf418f13b 100644
--- a/io.github.naikari.Naikari.metainfo.xml
+++ b/io.github.naikari.Naikari.metainfo.xml
@@ -42,8 +42,6 @@
     </screenshot>
   </screenshots>
 
-  <icon type="stock">naikari</icon>
-  
   <categories>
     <category>Game</category>
     <category>AdventureGame</category>

From 1d21daaf04b99712fe93d109635b858e7aae7c2b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matthias=20Mail=C3=A4nder?=
 <Mailaender@users.noreply.github.com>
Date: Sun, 26 Jun 2022 11:27:13 +0200
Subject: [PATCH 2/3] Add Open Age Rating

---
 io.github.naikari.Naikari.metainfo.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/io.github.naikari.Naikari.metainfo.xml b/io.github.naikari.Naikari.metainfo.xml
index dbf418f13b..56a30fe2f9 100644
--- a/io.github.naikari.Naikari.metainfo.xml
+++ b/io.github.naikari.Naikari.metainfo.xml
@@ -20,6 +20,8 @@
     </p>
   </description>
 
+  <content_rating type="oars-1.1" />
+
   <launchable type="desktop-id">io.github.naikari.Naikari.desktop</launchable>
   <screenshots>
     <screenshot type="default">

From 4457599091c84c6d1e1e3b9dc61734a16ece7fec Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matthias=20Mail=C3=A4nder?=
 <Mailaender@users.noreply.github.com>
Date: Sun, 26 Jun 2022 11:32:23 +0200
Subject: [PATCH 3/3] Add stable releases

---
 io.github.naikari.Naikari.metainfo.xml | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/io.github.naikari.Naikari.metainfo.xml b/io.github.naikari.Naikari.metainfo.xml
index 56a30fe2f9..2bd6ed7520 100644
--- a/io.github.naikari.Naikari.metainfo.xml
+++ b/io.github.naikari.Naikari.metainfo.xml
@@ -44,6 +44,27 @@
     </screenshot>
   </screenshots>
 
+  <releases>
+    <release version="v0.3.0" date="2022-06-09" type="stable">
+      <url>https://github.com/naikari/naikari/releases/tag/v0.3.0</url>
+    </release>
+    <release version="v0.2.1" date="2022-03-26" type="stable">
+      <url>https://github.com/naikari/naikari/releases/tag/v0.2.1</url>
+    </release>
+    <release version="v0.2.0" date="2022-03-13" type="stable">
+      <url>https://github.com/naikari/naikari/releases/tag/v0.2.0</url>
+    </release>
+    <release version="v0.1.3" date="2022-03-01" type="stable">
+      <url>https://github.com/naikari/naikari/releases/tag/v0.1.3</url>
+    </release>
+    <release version="v0.1.2" date="2022-02-22" type="stable">
+      <url>https://github.com/naikari/naikari/releases/tag/v0.1.2</url>
+    </release>
+    <release version="v0.1.0" date="2022-02-08" type="stable">
+      <url>https://github.com/naikari/naikari/releases/tag/v0.1.0</url>
+    </release>
+  </releases>
+
   <categories>
     <category>Game</category>
     <category>AdventureGame</category>
